### PR TITLE
Updated package name, function name in README.rst cdoe example

### PR DESCRIPTION
**Pull request by romilly**
The code example in  README.rst was only partially updated when the API was changed.

### CODE REVIEW

**Comment by dherrada** [comment]
@ladyada After #16 is merged, I think this pr should pass

**Comment by ladyada** [comment]
@romilly can you 'bump' this PR?

**Comment by dherrada** [comment]
@romilly I think what you've got to do is merge adafruit/master into your fork. I'm not super good at git, but I think you've got to add the adafruit fork as a remote, then fetch adafruit/master, then merge it into romilly/master

**Comment by kmatch98** [comment]
The latest edit uses an “out of date” solution for the “streak problem”. The solution that was chosen to merge was done in this issue: https://github.com/adafruit/Adafruit_CircuitPython_BitmapSaver/pull/14.  It prevents the buffer from being reallocated. 

Can you update your merge to match the released version?

**Comment by jposada202020** [comment]
@romilly as @kmatch98 mentioned this was solved with an improved solution. Could you modify the current PR to modify the Readme.rst file only? let me know thanks

File: adafruit_bitmapsaver.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def _write_pixels(output_file, pixel_source, palette):
                     color >>= 8
                     buffer_index += 1
         else:
+            result_buffer = bytearray(2048)
```
Comment (lesamouraipourpre) [inline adafruit_bitmapsaver.py:94]: The `result_buffer` line at Line 82 could be removed as it is no longer needed.

**Review by lesamouraipourpre — COMMENTED** [review]
Looks good to me.

File: adafruit_bitmapsaver.py (hunk 1 above)
Comment (FoamyGuy) [reply adafruit_bitmapsaver.py:93]: Thanks for the review @lesamouraipourpre. I removed that unused copy of the variable with latest commit

**Review by FoamyGuy — APPROVED** [review]
Looks good to me
tested successfully with:
```
Adafruit CircuitPython 7.1.0-beta.0 on 2021-11-12; Adafruit PyPortal with samd51j20
Board ID:pyportal
```

**Pull request merged**